.github/workflows/ci.yml

```
@@ -20,11 +20,18 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
+        with:
+          profile: minimal
```

actionlint (1.7.7)

input "profile" is not defined in action "dtolnay/rust-toolchain@stable". available inputs are "components", "target", "targets", "toolchain"

(action)